### DISCUSSION TITLE
What is /mta good for?

### DISCUSSION THREAD

**Discussion by bcalco** [leppie/IronScheme#103] [Discussions]
@leppie... I know the option to run in Multi-Threaded Apartment mode is part of the DLR (read: pain), but as I'm wrapping the console host I'm having to decide if I accept and use the `/mta` option that comes with the `ConsoleHost` abstraction. I get in concept what MTA means and is for, but see odd behavior when I enable it and am in general not sure what the win is, or in what context a win might be obtained by allowing the `/mta` option (I suspect for using GUIs like forms and such). Any examples would be appreciated.

**Comment by leppie** [comment]
That is for COM stuff. I have no idea what it does. I have never touched that explicitly.

**Reply by bcalco** [reply]
Hmm. Yes, I recall apartment choices for COM servers being a thing now. Sounds like something I should hide, then... at least until I can find a good use for it (maybe writing COM servers in Scheme, which feels.... kinda retro...but you know someone will ask how to do it...). LOL.
